developer-experience/readme.md

```diff
@@ -85,6 +85,12 @@ New team members are uniquely positioned to identify instances of undocumented [
 
 The following outline examples of several strategies that can be adopted to promote a positive developer experience. It is expected that each team should define what a positive dev experience means within the context of their project. Additionally, refine that over time via feedback mechanisms such as sprint and project retrospectives.
 
+### Observability
+
+[Observability](../observability/readme.md) alleviates unforeseen challenges for the developer in a complex distributed system. It identifies project bottlenecks quicker and with more precision, enhancing performance as the developer seeks to deploy code changes. Adding observability improves the experience when identifying and resolving bugs or broken code. This positive DevEx results in fewer or less severe current and future production failures.

```

Comment:
Would edit the last sentence a little to remove the words "positive DevEx" as it sounds a bit funny eg: "This also results in fewer or less severe....".

Reply:
+1